src/libraries/System.Private.CoreLib/src/System/RuntimeType.cs

```diff
@@ -72,6 +72,7 @@ public override IList<CustomAttributeData> GetCustomAttributesData()
 
         // GetDefaultMembers
         // This will return a MemberInfo that has been marked with the [DefaultMemberAttribute]
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
Why All? The doc says it can be only `constructor, method, property, or field` and I think they have to be public

Reply:
> The doc says it can be only constructor, method, property, or field and I think they have to be public

Couldn't find that mention in the doc - the impl also works for events and nested types. It's a janky API.

It was forced by the annotation on `GetMember`. I've fixed that up.